pyteal/ast/binaryexpr.py

```diff
@@ -259,3 +259,185 @@ def GetByte(value: Expr, index: Expr) -> BinaryExpr:
         index: The index of the byte to extract. Must evaluate to an integer less than Len(value).
     """
     return BinaryExpr(Op.getbyte, (TealType.bytes, TealType.uint64), TealType.uint64, value, index)
+
+def BAdd(left: Expr, right: Expr) -> BinaryExpr:
```

Comment:
I just used `B<name of op>` for the byteslice math ops, but it might not be explicit enough -- Thoughts on changing this to `Byte<name-of-op>`, e.g. `ByteAdd`?

Reply:
I think I prefer `Bytes<name-of-op>`

Reply:
Changed to `Bytes<name-of-op>`